server/services/WeekPhsaseService/weekPhasesService.js

```diff
@@ -0,0 +1,39 @@
+"use strict"
+
+var cron = require('node-cron');
+
+const {
+  getTrueActivePhase,
+  setPhaseIDOverride,
+  getCurrentPhaseID
+} = require("./weekPhasesFunctions");
+
+var phaseIDOverride = null;
+var activePhase = getTrueActivePhase();
+
+activePhase.handler();
+
+let cronTask = cron.schedule('* * * * *', ()=> {
+  let phase = getTrueActivePhase();
+
+  if(phase === null){
+    console.log("Currently no phase");
+    activePhase = null;
+    return;
+  }
+  if(phase.id !== activePhase.id){
+    console.log("New phase detected!");
+    activePhase = phase;
+    activePhase.handler();
+  }
+  else
+    console.log("No new phase");
+});
+
+cronTask.start();
+
+exports.getCurrentPhaseID = () => getCurrentPhaseID(activePhase, phaseIDOverride);
```

Comment:
Handlers are still not implemented and there's no way to set or get the phase from the extern. 
You didn't implement the apis as required.